ehcache-impl/src/main/java/org/ehcache/config/builders/UserManagedCacheBuilder.java

```diff
@@ -19,25 +19,21 @@
 import org.ehcache.Cache;
 import org.ehcache.CachePersistenceException;
 import org.ehcache.UserManagedCache;
-import org.ehcache.config.Builder;
-import org.ehcache.config.CacheConfiguration;
-import org.ehcache.config.EvictionAdvisor;
-import org.ehcache.config.ResourcePools;
-import org.ehcache.config.ResourceType;
+import org.ehcache.config.*;
```

Comment:
Looks like the IDE did some undesired magic here. We should put these back.

Reply:
reverted back all import changes